### PR TITLE
Bumped to React 16, removed reactCompat

### PR DESCRIPTION
**Pull request by nicorobo**
I don't want this merged until I have similar updates applied to Hermes and can confirm they work together. Putting that together now!

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -21,21 +21,17 @@
...
-    "react": "^15.5.4",
-    "react-dom": "^15.5.4"
+    "mochify": "^2.18.0"
   },
   "peerDependencies": {
-    "react": ">0.12.0"
-  },
-  "dependencies": {
-    "@streammedev/react-compat": "0.0.0"
+    "react": "^16.0.0",
```
Comment (wesleytodd) [inline package.json]: Change these to just `>15.0.0`.  I think anything above that should support the `React.Component` class extension.

File: src/index.jsx [hunk 2]
```diff
@@ -1,8 +1,24 @@
-import React from 'react';
-import reactCompat from '@streammedev/react-compat';
-import {FlyoutToggle} from './toggle';
+const React = require('react');
+const PropTypes = require('prop-types');
+const {FlyoutToggle} = require('./toggle');
```
Comment (wesleytodd) [inline src/index.jsx]: Does this need to be eestructured now that you removed the import/exports?

**Review by wesleytodd — APPROVED** [review]
After those changes, approved!

File: src/index.jsx (hunk 2 above)
Comment (nicorobo) [reply src/index.jsx]: Yes it does, thanks!

File: package.json [hunk 3]
```diff
@@ -30,8 +35,8 @@
     "mochify": "^2.18.0"
   },
   "peerDependencies": {
-    "react": "^16.0.0",
-    "react-dom": "^16.0.0",
+    "react": ">15.0.0",
+    "react-dom": ">15.0.0",
     "prop-types": "^15.5.10"
```
Comment (wesleytodd) [inline package.json]: This should also be greater than.

**Comment by nicorobo** [comment]
It seems like I can't publish a beta to test until I'm added as a contributor, can whoever sees this first add me? Thanks!

**Pull request merged**